Makefile

```diff
@@ -48,6 +48,10 @@ build-metadata:
 binary: modcheck swagger create-name build-metadata
 	env CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o bin/zot-$(OS)-$(ARCH) -buildmode=pie -tags $(EXTENSIONS),containers_image_openpgp -v -trimpath -ldflags "-X zotregistry.io/zot/pkg/api/config.Commit=${COMMIT} -X zotregistry.io/zot/pkg/api/config.BinaryType=$(extended-name) -X zotregistry.io/zot/pkg/api/config.GoVersion=${GO_VERSION} -s -w" ./cmd/zot
 
+.PHONY: binary-graphqlplayground
```

Comment:
just call this binary-gql?

Reply:
That would imply it's for the search. And if we later decide rename the 'search' endpoint to 'graphql' (since it would include mutations for toggling bookmarks) it would be confusing.